packages/amplify-category-xr/lib/xr-manager.js

```diff
@@ -139,7 +139,7 @@ async function addSceneConfig(context, sceneName) {
     validate: (configFilePath) => {
       try {
         if (fs.existsSync(configFilePath)) {
-          sumerianConfig = require(configFilePath);
+          sumerianConfig = JSON.parse(fs.readFileSync(configFilePath));
```

Comment:
we should use the util function from this PR https://github.com/aws-amplify/amplify-cli/pull/1293 to read JSON data

Reply:
Done